pkg/ccloudv2/provider_integration_v2.go

```diff
@@ -28,41 +28,47 @@ func (c *Client) V2ApiContext(ctx context.Context) context.Context {
 
 // CreatePimV2Integration creates a new provider integration
 func (c *Client) CreatePimV2Integration(ctx context.Context, request piv2.PimV2Integration) (piv2.PimV2Integration, error) {
-	integration, _, err := c.ProviderIntegrationV2Client.IntegrationsPimV2Api.CreatePimV2Integration(c.V2ApiContext(ctx)).PimV2Integration(request).Execute()
+	integration, httpResp, err := c.ProviderIntegrationV2Client.IntegrationsPimV2Api.CreatePimV2Integration(c.V2ApiContext(ctx)).PimV2Integration(request).Execute()
 	if err != nil {
-		return piv2.PimV2Integration{}, err
+		return piv2.PimV2Integration{}, errors.CatchCCloudV2Error(err, httpResp)
```

Comment:
Haven't worked in CLI repo for a while so double checking, this file is not auto-generated or copy-pasted from somewhere right? Otherwise, we might forget these changes and accidentally overwrite in the future.

Reply:
It's not auto generated.